### PR TITLE
Closes 38 Remove duplicate data type handlers from the configured lis…

### PR DESCRIPTION
**Pull request by friedlou**
…t of data type handlers.

### CODE REVIEW

File: warehouse/core/src/main/java/datawave/util/StringUtils.java [hunk 1]
```diff
@@ -1,10 +1,6 @@
 package datawave.util;
 
-import java.util.ArrayList;
-import java.util.Arrays;
-import java.util.Iterator;
-import java.util.List;
-import java.util.NoSuchElementException;
+import java.util.*;
```
Comment (brianloss) [inline warehouse/core/src/main/java/datawave/util/StringUtils.java]: Don't import with wildcards, please.

Comment (friedlou) [reply warehouse/core/src/main/java/datawave/util/StringUtils.java]: Funny. I did not make that change deliberately. Most probably Intelij did it. I will fix it.

File: warehouse/core/src/main/java/datawave/util/StringUtils.java [hunk 2]
```diff
@@ -429,4 +431,22 @@ static boolean isEscapableLiteral(char c) {
             return values;
         }
     }
+    
+    /**
+     * Remove the strings that are empty
```
Comment (brianloss) [inline warehouse/core/src/main/java/datawave/util/StringUtils.java]: Documentation of this method is incorrect.

**Comment by brianloss** [comment]
Your commit comment should indicate it "closes" or "fixes" 38, so that the associated issue will automatically be closed when the PR is merged.

**Comment by friedlou** [comment]
done

**Pull request merged**